tests/functional/test_install_vcs.py

```diff
@@ -241,21 +241,6 @@ def test_git_with_ambiguous_revs(script):
 
 
 @pytest.mark.network
-def test_git_works_with_editable_non_origin_repo(script):
```

Comment:
Why has this test been removed?

If there's a behaviour change, it would be wiser to update this test.

Reply:
Because this test is asserting the (IMO) broken behavior that an error is raised if using an editable install against git repo with no remotes. I updated the test that asserts proper behavior to include the no git remote situation.

This shouldn't be an error (IMO) and therefore we shouldn't have a test asserting that there is an error.

Reply:
Cool. :)